app/lib/frontend/backend.dart

```diff
@@ -534,6 +536,12 @@ class GCloudPackageRepository extends PackageRepository {
       ),
     );
 
+    // Trigger analysis and dartdoc generation
+    await analyzerClient.triggerAnalysis(
+        newVersion.package, newVersion.version, <String>{});
```

Comment:
Please document that you left out dependent packages as this will be done later by a background process.